runtime/src/lib.rs

```diff
@@ -308,23 +307,51 @@ parameter_types! {
 	pub const SS58Prefix: u16 = 2021;
 }
 
+pub struct BaseCallFilter;
+impl Contains<RuntimeCall> for BaseCallFilter {
+	fn contains(call: &RuntimeCall) -> bool {
+		match call {
+			RuntimeCall::Recovery(pallet_recovery::Call::create_recovery{ .. }) => false,
+			RuntimeCall::Multisig(pallet_multisig::Call::approve_as_multi{ .. }) => false,
+			RuntimeCall::Multisig(pallet_multisig::Call::as_multi{ .. }) => false,
+			_ => true
+		}
+	}
+}
+
 /// The default types are being injected by [`derive_impl`](`frame_support::derive_impl`) from
 /// [`ParaChainDefaultConfig`](`struct@frame_system::config_preludes::ParaChainDefaultConfig`),
 /// but overridden as needed.
 #[derive_impl(frame_system::config_preludes::ParaChainDefaultConfig as frame_system::DefaultConfig)]
 impl frame_system::Config for Runtime {
+	/// The basic call filter to use in dispatchable.
+	type BaseCallFilter = BaseCallFilter;
 	/// The identifier used to distinguish between accounts.
 	type AccountId = AccountId;
+	/// The aggregated dispatch type that is available for extrinsics.
+	type RuntimeCall = RuntimeCall;
+	/// The lookup mechanism to get account ID from whatever is passed in dispatchers.
+	type Lookup = AccountIdLookup<AccountId, ()>;
```

Comment:
I think that `RuntimeCall` and `Lookup` are automatically set by `derive_impl`.

Reply:
Indeed.